CHANGELOG.md

```diff
@@ -69,6 +69,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Support OpenSSL Provider with default Netty allocator ([#5460](https://github.com/opensearch-project/OpenSearch/pull/5460))
 - Replaces ZipInputStream with ZipFile to fix Zip Slip vulnerability ([#7230](https://github.com/opensearch-project/OpenSearch/pull/7230))
 - Add missing validation/parsing of SearchBackpressureMode of SearchBackpressureSettings ([#7541](https://github.com/opensearch-project/OpenSearch/pull/7541))
+- Change Aggregator Factories to return null if empty aggs body is present in search request ([#8206](https://github.com/opensearch-project/OpenSearch/pull/8206))
```

Comment:
Two things:

- This fixes a bug for an unreleased change, so we don't need a changelog entry as this bug was never visible in a release.
- Not relevant here, but for future reference, any change that is intended to be backported to 2.x should be in the `[Unreleased 2.x]` section of the changelog. See [CONTRIBUTING.md](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#changelog) for comprehensive guidance on the changelog.